persistence/src/main/java/de/cyface/persistence/serialization/AccelerationsFile.java

```diff
@@ -0,0 +1,81 @@
+package de.cyface.persistence.serialization;
+
+import java.io.BufferedOutputStream;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.util.List;
+
+import android.content.Context;
+
+import de.cyface.persistence.FileUtils;
+import de.cyface.persistence.model.Point3D;
+
+/**
+ * The file format to persist the captured acceleration points.
+ *
+ * @author Armin Schnabel
+ * @version 1.0.0
+ * @since 3.0.0
+ */
+public class AccelerationsFile implements FileSupport<List<Point3D>> {
```

Comment:
Why are there classes for all the different sensors. Shouldn't one class be enough? They behave equally except for the location they store the data to, so this difference should be solvable using parameters.

Reply:
True. taking care of it right now...